cpp/oneapi/dal/detail/common.cpp

```diff
@@ -33,7 +33,8 @@ void integer_overflow_ops<Data>::check_mul_overflow(const Data& first, const Dat
         volatile Data tmp = first * second;
         tmp /= first;
         if (tmp != second) {
-            throw range_error("overflow found in multiplication of two values");
+            throw range_error(
+                dal::detail::error_messages::overflow_found_in_multiplication_of_two_values());
```

Comment:
Shouldn't it be named `overfow_found_in_mul_of_two_values` for consistency with function above?

Reply:
Will rename in next PR